doc/faq.txt

```diff
@@ -48,7 +48,8 @@ variable to a different value. Please consult your OS's documentation to learn
 how to configure these variables in detail.
 
 If you want to apply the same configuration every time you start yash, write
-the command in ~/.yashrc or ~/.yash_profile.
+the command in ~/.config/yashrc or ~/.config/yash_profile. You can also write
+to ~/.yashrc or ~/.yash_profile.
```

Comment:
Since we are not going to support loading both rc files at the same time, I think it would be clearer to write:

```suggestion
the command in ~/.config/yashrc or ~/.config/yash_profile. The old locations
~/.yashrc and ~/.yash_profile are also supported as a fallback.
```